pkg/yaml/yaml.go

```diff
@@ -32,20 +33,31 @@ func ParseObjects(manifest string) ([]*unstructured.Unstructured, error) {
 	r := strings.NewReader(manifest)
 	decoder := yaml.NewYAMLReader(bufio.NewReader(r))
 	ret := []runtime.Object{}
+	nullResult := []byte("null")
```

Comment:
any reason to have this as a variable? I think it's more readable to have `bytes.Equal(jsondata, []byte("null"))` and we don't reuse this.

Reply:
The only reason is to define the variable only once outside of the for loop. It is a minor optimization.

Happy to change it though.

Reply:
I just noticed that if anything it should be a const.

Reply:
:+1: that sounds like a good reason, I didn't have a strong opinion so let's leave it.